locales/pt/messages.po

```diff
@@ -0,0 +1,1423 @@
+"voluntários que possa tratar da revisão de propostas, gerar e verificar "
+"legendas, apresentar palestras e traduzir o conteúdo nas nossas plataformas."
+
+#: src/components/faq.astro:26
+msgid "I don't know Python, but I'm interested into learning, should I join?"
+msgstr "Não sei Python, mas tenho interesse em aprender, devo participar?"
+
+#: src/components/faq.astro:27
+msgid ""
+"Some topics might be difficult to understand, but we aim to have talks from "
+"many levels, including beginner topics, and also some activities for people "
+"to learn."
+msgstr ""
+"Alguns tópicos podem ser difíceis de entender, mas pretendemos ter palestras "
+"de vários níveis, incluindo tópicos para iniciantes, e também algumas "
+"atividades para as pessoas aprenderem."
+
+#: src/components/faq.astro:30
+msgid "Do you need help with the conference?"
+msgstr "Precisam de ajuda com a conferência?"
+
+#: src/components/faq.astro:31
+msgid ""
+"Yes!, we are usually looking for volunteers that can help us in tasks like "
+"reviewing proposals, generate and check subtitles, moderate sprints/"
+"workshops/panels, moderate discord, help us with the website, and many other "
+"things."
+msgstr ""
+"Sim!, geralmente procuramos voluntários que nos possam ajudar em tarefas como "
+"rever propostas, gerar e verificar legendas, moderar sprints/workshops/"
+"painéis, moderar o discord, ajudar-nos com o site e muitas outras coisas."
+
+#: src/components/faq.astro:38
+msgid "Frequently Asked Questions"
+msgstr "Perguntas Frequentes"
+
+#: src/components/hero.astro:42
+msgid "PyLadiesCon 2025"
+msgstr "PyLadiesCon 2025"
+
+#: src/components/hero.astro:44
+msgid "5th-7th December | Multi-language | Multi-timezone"
+msgstr "5-7 de Dezembro | Multilíngue | Multi-fuso horário"
+
+#: src/components/hero.astro:65 src/data/menuItems.ts:20
+#: src/pages/[lang]/schedule.astro:22
+msgid "Schedule"
+msgstr "Horário"
+
+#: src/components/hero.astro:73
+msgid "Register"
+msgstr "Registar"
+
+#: src/components/hero.astro:77 src/data/menuItems.ts:43
+#: src/pages/[lang]/donate.astro:16
+msgid "Donate"
+msgstr "Doar"
+
+#: src/components/hero.astro:80
+msgid "Visit our meta website"
+msgstr "Visita o nosso site meta"
+
+#: src/components/hero.astro:93
+msgid "The online conference of the PyLadies communities around the globe."
+msgstr "A conferência online das comunidades PyLadies de todo o mundo."
+
+#: src/components/keynotes_index.astro:21
+msgid "Meet Our Keynotes"
+msgstr "Conhece os nossos Keynotes"
+
+#: src/components/keynotes_index.astro:22 src/data/menuItems.ts:6
+msgid "Keynotes"
+msgstr "Keynotes"
+
+#: src/components/organization.astro:14
+msgid ""
+"Mariatta is a Python core developer where she focuses on improving the "
+"workflow and documentation. She is active in the Python community as an "
+"advisor for the Global PyLadies, and PyCon US Chair."
+msgstr ""
+"A Mariatta é uma developer de Python core que se dedica a melhorar o fluxo de "
```

Comment:
I don't know if the PT-speaking community has a nice term for “Python Core Developer” but I think “developer de Python core” doesn't convey the right idea in PT.